### PR TITLE
added ourmission grid

### PR DESCRIPTION
**Pull request by andrew3521**
Screenshot 2024-07-08 at 3 32 06 PM

### CODE REVIEW

**Review by stanleylew5 — CHANGES_REQUESTED** [review]
add a line break after

The X in womxn honors women’s intersecting identities, as women in America are impacted by a multitude of social justice and human rights issues. - Kimberlé Crenshaw

also try and get the image's height to be the same height as the text 

please pull from your own branch before you start coding

git pull origin andrew3521/mission

**Comment by andrew3521** [comment]
Screenshot 2024-07-08 at 8 28 44 PM
Screenshot 2024-07-08 at 8 28 44 PM

File: src/components/about/OurMission.jsx [hunk 1]
```diff
@@ -1,7 +1,32 @@
...
+            accomplished attorneys, Womxn in the Law amplifies the needs of
+            UCR’s most disadvantaged student population.
+          </p>
+          <p>
+            The X in womxn honors women’s intersecting identities, as women in
+            America are impacted by a multitude of social justice and human
+            rights issues. - Kimberlé Crenshaw
+          </p>
+        </div>
+        <div className="pl-52 pt-8">
```
Comment (stanleylew5) [inline src/components/about/OurMission.jsx]: let's not hard code values like pl-52 use flex and justify-end and then add like padding values of four

File: src/components/about/OurMission.jsx [hunk 2]
```diff
@@ -1,7 +1,32 @@
+import Pic from "@/public/about/grouppic2.webp";
+import Image from "next/image";
 const OurMission = () => {
   return (
-    <div className="flex flex-row">
-      <p className="m-auto p-auto text-7xl">Our Mission</p>
+    <div>
```
Comment (stanleylew5) [inline src/components/about/OurMission.jsx:5]: add some padding beneath your div like pb-2

File: src/components/Dummy.jsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+          Womxn In The Law
+        </div>
+        <div className="bg-witl-purple-300 text-white w-fit text-8xl p-12 col-span-2">
+          Womxn In The Law
+        </div>
+      </div>
+    </div>
+  );
+};
+export default Dummy;
```
Comment (stanleylew5) [inline src/components/Dummy.jsx]: delete this file

**Review by stanleylew5 — CHANGES_REQUESTED** [review]

**Comment by andrew3521** [comment]
image

**Pull request merged**